### PR TITLE
Getting started tutorials

### PR DESCRIPTION
**Pull request by natashadecoste**
### Updates
- change storybook tutorial structure
- tutorials are exported from their own files

### Review checklist (for reviewers only)
- [ ] Demos all features
- [ ] Documented/annotated
- [ ] Matches UI/UX specs
- [ ] Meets the [code style guide](https://github.com/IBM/carbon-charts/wiki/Code-style-guide)
- [ ] Accessible
- [ ] Mobile first (responsive)
- [ ] RTL support (bidirectional text)
- [ ] Performant (limited bloat)

### CODE REVIEW

File: packages/core/stories/tutorials/getting-started/angular.ts [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+**with npm:**
+\`\`\`bash
+npm install --save @carbon/charts @carbon/charts-angular d3
+\`\`\`
+
+&nbsp;
+## Getting started
+
+To start using the \`BarChart\` component, try the example below:
```
Comment (theiliad) [inline packages/core/stories/tutorials/getting-started/angular.ts:23]:
```suggestion
To start using the \`StackedBarChart\` component, try the example below:
```

File: packages/core/stories/tutorials/getting-started/angular.ts [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+&nbsp;
+## Data and Options
+Data and options follow the same model in all charts, with minor exceptions and differences in specific components.
+See tutorial on tabular data format for more information on chart data.
+
+&nbsp;
+## Demos and Examples
++ **[Storybook - Demos](https://carbon-design-system.github.io/carbon-charts/angular)**
++ **[Storybook - Story Sources](https://github.com/carbon-design-system/carbon-charts/tree/master/packages/angular/stories)**
++ **[Storybook - Data & options used](https://github.com/carbon-design-system/carbon-charts/tree/master/packages/core/demo/demo-data)**
```
Comment (theiliad) [inline packages/core/stories/tutorials/getting-started/angular.ts:83]:
```suggestion
+ **[Storybook - Data & options used](https://github.com/carbon-design-system/carbon-charts/tree/master/packages/core/demo/data)**
```

File: packages/core/stories/tutorials/getting-started/react.ts [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+&nbsp;
+## Data and Options
+Data and options follow the same model in all charts, with minor exceptions and differences in specific components.
+See tutorial on tabular data format for more information on chart data.
+
+&nbsp;
+## Demos and Examples
++ **[Storybook - Demos](https://carbon-design-system.github.io/carbon-charts/react)**
++ **[Storybook - Story Sources](https://github.com/carbon-design-system/carbon-charts/tree/master/packages/react/stories)**
++ **[Storybook - Data & options used](https://github.com/carbon-design-system/carbon-charts/tree/master/packages/core/demo/demo-data)**
```
Comment (theiliad) [inline packages/core/stories/tutorials/getting-started/react.ts:69]:
```suggestion
+ **[Storybook - Data & options used](https://github.com/carbon-design-system/carbon-charts/tree/master/packages/core/demo/data)**
```

File: packages/core/stories/tutorials/getting-started/vanilla.ts [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+\`\`\`
+
+&nbsp;
+## Data and Options
+Data and options follow the same model in all charts, with minor exceptions and differences in specific components.
+See tutorial on tabular data format for more information on chart data.
+
+&nbsp;
+## Demos and Examples
++ **[Demos - Data & options used](https://github.com/carbon-design-system/carbon-charts/tree/master/packages/core/demo/demo-data)**
```
Comment (theiliad) [inline packages/core/stories/tutorials/getting-started/vanilla.ts:93]:
```suggestion
+ **[Demos - Data & options used](https://github.com/carbon-design-system/carbon-charts/tree/master/packages/core/demo/data)**
```

File: packages/core/stories/tutorials/getting-started/vue.ts [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+&nbsp;
+## Data and Options
+Data and options follow the same model in all charts, with minor exceptions and differences in specific components.
+See tutorial on tabular data format for more information on chart data.
+
+&nbsp;
+## Demos and Examples
++ **[Storybook - Demos](https://carbon-design-system.github.io/carbon-charts/vue)**
++ **[Storybook - Story Sources](https://github.com/carbon-design-system/carbon-charts/tree/master/packages/vue/stories)**
++ **[Storybook - Data & options used](https://github.com/carbon-design-system/carbon-charts/tree/master/packages/core/demo/demo-data)**
```
Comment (theiliad) [inline packages/core/stories/tutorials/getting-started/vue.ts:63]:
```suggestion
+ **[Storybook - Data & options used](https://github.com/carbon-design-system/carbon-charts/tree/master/packages/core/demo/data)**
```

File: packages/core/stories/tutorials/getting-started/vanilla.ts [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+**with npm:**
+
+\`\`\`bash
+npm install --save @carbon/charts d3
+\`\`\`
+
+&nbsp;
+## Getting started
+
+To start using the \`BarChart\` component, try the example below:
```
Comment (theiliad) [inline packages/core/stories/tutorials/getting-started/vanilla.ts:24]:
```suggestion
To start using the \`StackedBarChart\` component, try the example below:
```

File: packages/core/stories/tutorials/getting-started/angular.ts [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+\`\`\`ts
+import { Component } from "@angular/core";
+
+@Component({
+  selector: "my-app",
+  templateUrl: "./app.component.html",
+  styleUrls: ["./app.component.css"],
+})
+export class AppComponent {
+  stackedBarData = {
```
Comment (theiliad) [inline packages/core/stories/tutorials/getting-started/angular.ts:64]:
```suggestion
  stackedBarData = [
    // ...see next section
  ];
```

File: packages/core/stories/tutorials/getting-started/angular.ts [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+    // ...see next section
```
Comment (theiliad) [inline packages/core/stories/tutorials/getting-started/angular.ts:65]:
```suggestion
```

File: packages/core/stories/tutorials/getting-started/angular.ts [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+  };
```
Comment (theiliad) [inline packages/core/stories/tutorials/getting-started/angular.ts:66]:
```suggestion
```

File: packages/core/stories/tutorials/getting-started/angular.ts [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+
```
Comment (theiliad) [inline packages/core/stories/tutorials/getting-started/angular.ts:67]: same as above for all the other packages^^

Comment (natashadecoste) [reply packages/core/stories/tutorials/getting-started/angular.ts:67]: should i just leave it empty?
` stackedBarData = [];`
or should i say to see the tabular format tutorial or something?

Comment (theiliad) [reply packages/core/stories/tutorials/getting-started/angular.ts:67]: yes point to the tabular format

**Comment by theiliad** [comment]
> Are we keeping lines to a certain length?
> There are some loooong was in markdown strings and we should be able to split them almost wherever.

these are markdown syntax, that seems unnecessary.

this isn't code... it's tutorials

A better fix would be to actually turn these into `.md` files.

we need these tutorials out immediately

**Comment by zvonimirfras** [comment]
> > Are we keeping lines to a certain length?
> > There are some loooong was in markdown strings and we should be able to split them almost wherever.
> 
> these are markdown syntax, that seems unnecessary.
> 
> this isn't code... it's tutorials
> 
> A better fix would be to actually turn these into `.md` files.
> 
> we need these tutorials out asap

Splitting lines into readable length is not because it's necessary for code to work, it's to make the code more readable. Since it's markdown, splitting is really easy since it doesn't affect the functionality. It's really easier to do than to talk about.

**Pull request merged**